### ISSUE TITLE
Crashes when someone links to r/alien-blue

### ISSUE THREAD

**Issue by SlideAutoreport** [ccrama/Slide#240]
someone linked alien-blue in the OP. i click on it  says Subreddit not Found then it crashes

Debug info:
 OS Version: 3.10.9-talexop-1.1(N910CXXU1COI6)
 OS API Level: 22
 Device: trelte
 Model (and Product): SM-N920C (treltexx)
 RELEASE: 5.1.1
 BRAND: samsung
 DISPLAY: LMY47X.N910CXXU1COI6
 CPU_ABI: armeabi-v7a
 CPU_ABI2: armeabi
 HARDWARE: universal5433
 MANUFACTURER: samsung

Stacktrace: 

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{me.ccrama.redditslide/me.ccrama.redditslide.Activities.MultiredditOverview}: java.lang.NullPointerException: Attempt to invoke interface method 'int java.util.List.size()' on a null object reference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3149)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3248)
	at android.app.ActivityThread.access$1000(ActivityThread.java:197)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1681)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:145)
	at android.app.ActivityThread.main(ActivityThread.java:6872)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199)
	at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:115)
Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'int java.util.List.size()' on a null object reference
	at me.ccrama.redditslide.Activities.MultiredditOverview.setDataSet(MultiredditOverview.java:197)
	at me.ccrama.redditslide.Activities.MultiredditOverview.onCreate(MultiredditOverview.java:72)
	at android.app.Activity.performCreate(Activity.java:6550)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1120)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3102)
	... 11 more
```

**Comment by ccrama** [comment]
Appears to be multireddit related, not opening a subreddit. Looking into it.

**Comment by KaliK0t** [comment]
https://www.reddit.com/r/announcements/comments/3p4shh/ceo_steve_here_to_answer_more_questions/

this is the link to where it crashes when i click on u/alien-blue

**Comment by ccrama** [comment]
Read my above comment

**Comment by KaliK0t** [comment]
Downgraded to play store version, clean install still happens on that post without going through multireddit

**Comment by xnaas** [comment]
Video of occurence: https://vid.me/lQ4u
Link causing the trouble: https://www.reddit.com/user/alien-blue

Note: Going directly to /u/alien-blue manually in the app works just fine.

**Comment by Alexendoo** [comment]
What appeared to happen for myself: the app attempts to load /r/alien-blue, a brief dialogue appears saying there is no such sub. Then it crashes.

**Comment by ccrama** [comment]
Fixed 92cbc617d420042e1a3d8c582d5f60e4f59148cb. I was only looking for /u/, not /user/!
